docs/source/tutorial/index.md

```diff
@@ -2,17 +2,16 @@
 
```

Comment:
I copied your text - except for the 'few more steps', because right now you need to comment parts of the pipeline if you don't want it to run the level_0 processing. There you actually need to go into the halodrops source code, which also involves a couple of steps.

Reply:
Good point!